app/static/css/scale.css

```diff
@@ -24,4 +24,30 @@
   background-color: #F0C5C5;
 }
 
+#sliderTwo .rs-handle  {

```

Comment:
Please add a comment to differentiate the code above with the code below

Reply:
added comments